examples/cards.rb

```diff
@@ -0,0 +1,32 @@
+$:.unshift File.expand_path("../lib", File.dirname(__FILE__))
+require 'lob.rb'
+require 'pp'
+
+# initialize Lob object
+lob = Lob::Client.new(api_key: 'YOUR_API_KEY')
```

Comment:
Let's update this just for some best practice reasons:
```suggestion
LOB_LIVE_API_KEY = ENV['LOB_LIVE_API_KEY']
lob = Lob::Client.new(api_key: LOB_LIVE_API_KEY)
```